src/vhdl/Makefile

```diff
@@ -79,31 +79,37 @@ vhdl-elocations_meta.adb: vhdl-elocations_meta.adb.in vhdl-elocations.ads $(DEPS
 ../../pyGHDL/libghdl/vhdl/nodes.py: $(DEPS) $(PNODESPY)
 	$(RM) $@
 	$(PNODESPY) $(NODES_FLAGS) libghdl-nodes > $@
+	python -m black $@
```

Comment:
I don't think this is desirable. Running `python -m black` or `black` will work. We can optionally provide a list of subdirs to include/exclude, as a target in the main makefile.

Reply:
The problem is that the generated files are set `chmod -w` in the next line, so they can't be formatted after being generated. Or are you suggesting not formatting these at all?